src/core/Chip/Chip.md

```diff
@@ -0,0 +1,30 @@
+```js
+import { Chip } from 'suomifi-ui-components';
+
+const removeAction = () => {
+  alert('Selection removed');
+};
+
+<>
+  <div styles={{ display: 'inline' }}>
+    <Chip removable actionLabel="Unselect" onClick={removeAction}>
+      Removable chip 1
+    </Chip>
+
+    <Chip removable actionLabel="Unselect" onClick={removeAction}>
+      Removable chip 2
+    </Chip>
+  </div>
+  <div>
+    <Chip onClick={removeAction}>
+      Clickable chip without remove icon
+    </Chip>
+  </div>
+
+  <Chip.static>
+    Static chip with a long content that exceeds the size limit
```

Comment:
This is a good example. Maybe even add the default max width to label? "exceeds the default maximum width of xx px." Since size limit is a bit ambiguous and could refer to character limit as well.

Reply:
Then I need to rephrase the content, since the max width of the content depends on the variant while the max width of the component stays constant (290px). I'd be quite happy keeping this as it is.

Reply:
I would rephrase so that it says that max-width of the component is 290px. It's relevant information and helps devs when documented. Maybe something like "Static chip with long content that does not fit into components max-width of 290px."

Reply:
Good suggestion. Put it into use.